classes/plugins/Plugin.inc.php

```diff
@@ -390,21 +390,41 @@ public function _overridePluginTemplates($hookName, $args) {
 		if (strpos($checkFilePath, $libPkpPrefix) === 0) $checkFilePath = substr($filePath, strlen($libPkpPrefix));
 
 		// Check if an overriding plugin exists in the plugin path.
-		$checkPluginPath = sprintf('%s/%s', $this->getPluginPath(), $checkFilePath);
-		if (file_exists($checkPluginPath)) {
-			$filePath = $checkPluginPath;
+		if ($this->_findOverriddenTemplate($checkFilePath)) {
+			$filePath = $this->_findOverriddenTemplate($checkFilePath);
```

Comment:
Just for a minor performance gain, can we do:

```php
if ($overriddenFilePath = $this->_findOverriddenTemplate($checkFilePath)) {
	$filePath = $overriddenFilePath;
}
```

Reply:
Thanks, changed!